src/redistributors/morpho/AccountFundsDistribution.sol

```diff
@@ -36,7 +36,7 @@ abstract contract AccountFundsDistribution {
 
         if (amount_ == 0) return;
 
-        _correctionPoints[vault_][account_] -= Common.toInt256(_pointsPerShares[vault_] * amount_); // _correctForReceipt
+        _correctionPoints[vault_][account_] -= Common.toInt256(_pointsPerShares[vault_] * amount_);
```

Comment:
Would be interesting to explain how the correction works.

Reply:
The other model (Scalable Rewards) tracks the last index for each account and computes the claimable using the difference between the current global index and the account's index. The downside of this is that when an account is poked, it's index is set to the current global index, so any rewards are "lost". So accounts need to take possession of their distributed rewards every time (on deposit and withdraw), or a new storage slot to track what has not yet been claimed, but has been "accounted for".

This model instead tracks a correction (denominated in points, which is just shares scaled by the PRECISION, so as to not lose precision) so that you can always just multiply the current global index by the shares (which gives you "lifetime" rewards of the current shares, since there is no index delta used), but then the correction has accounted for what should be:
- added to "lifetime" rewards due to shares that were previously withdrawn
- subtracted from "lifetime" rewards due to shares that were previously deposited

For example, if the current global index is 1,000,000, and you deposit your first 50 shares, the "lifetime" rewards of those shares (assuming their were deposited at global index 0), is 1,000,000 x 50. So to ensure those get nullified when computing the actual "lifetime" for a deposit _not made at global index 0, upon the deposit, the correction is set to -1,000,000 x 50. It banks what you are not entitled to.

If the current global index is 1,000,000, and you withdraw your 50 shares (that you despited at global index 0), the "lifetime" rewards of your remaining 0 shares is 1,000,000 x 0. So to ensure those withdrawn shares remain part of the actual "lifetime" after a withdrawal _made at global index 1,000,000, upon the withdrawal, the correction is set to +1,000,000 x 50. It banks what you are entitled to.

Correction is just a way to bank the lifetime effects of deposits and withdrawals made at specific global indexes, so that you can always determine the actual lifetime rewards for an account, without forcing them to "claim" right then and there.